hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotCreateRequest.java

```diff
@@ -225,6 +224,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     if (exception == null) {
       LOG.info("Created snapshot '{}' under path '{}'",
           snapshotName, snapshotPath);
+      omMetrics.incNumSnapshotActive();
```

Comment:
We should also `decNumSnapshotActive()` in `OMSnapshotDeleteRequest`

Reply:
Added. Thanks for the catch.